stratum/hal/lib/phal/onlp/onlpphal.cc

```diff
@@ -101,6 +101,10 @@ ::util::Status OnlpPhal::InitializePhalDB() {
   ASSIGN_OR_RETURN(std::move(database_),
                    AttributeDatabase::MakePhalDB(std::move(configurator)));
 
+  // Create and run PhalDb service
+  phal_db_service_ = absl::make_unique<PhalDbService>(database_.get());
+  phal_db_service_->Run();
+
```

Comment:
Hi Max, as per our discussion let's move the phaldb_service code into the MakePhalDB function in the attribute_database.cc code so that it is independent from the ONLP code.

It might also make sense to change the Adapter code (adapter & sfp_adapter) to take a pointer to the database rather than the phal_interface.

Reply:
Done.

I modified the adapters as part of the original PR already.